tests/integration/llm/prepare.py

```diff
@@ -538,6 +542,10 @@
         "true",
         "option.max_lora_rank":
         64,
+        "option.fully_sharded_loras":
+        "true",
+        "option.long_lora_scaling_factors":
+        "(4.0, )",
```

Comment:
Why does it have to be a strict tuple? Can we get comma separated strings and convert into tuple internally? This is what we do for context_length_estimate for tnx.

Reply:
Changed. Now we can get comma separated strings and convert into tuple.